src/rust-util/pelikan-sys/build.rs

```diff
@@ -295,6 +295,26 @@ fn main() {
             .write_to_file(out_path.join("ds_sarray.rs"))
             .expect("Couldn't write bindings");
     }
+    if cfg!(feature = "ds_smap") {
+        print_directives("ds_smap", "data_structure/smap");
+
+        let bindings = builder()
+            .header("../../data_structure/smap/smap.h")
+            .whitelist_type("smap_p")
+            .whitelist_type("smap_rstatus_e")
+            .whitelist_function("smap_.*")
+            .whitelist_recursively(false)
+            .derive_default(true)
+            .derive_copy(true)
+            .derive_debug(true)
```

Comment:
This is already handled by the `builder` function. It's better if we don't override this - especially if we want to change the defaults at a later time. (e.g. one of the PRs I'm working on does exactly this)